jolin-azeus/superset
# PoC of X-chart Control Chart

### SUMMARY
Added custom visualization named X-chart Control Chart

### TESTING INSTRUCTIONS
See
xcontrol_poc/How to use Xcontrol Chart in Superset.docx
xcontrol_poc/sample.sql
